### ISSUE TITLE
Accelerate on Intel Macs

### ISSUE THREAD

**Issue by jafri** [ggerganov/whisper.cpp#118]
The accelerate framework should be available on intel macs too I believe

**Comment by ggerganov** [comment]
Yes - it is available. I need to update the `Makefile` to support it.
But I think using `CMake` will already support it if you want to give it a try.
